backend/tracim_backend/lib/utils/request.py

```diff
@@ -255,6 +260,11 @@ def set_user(self, user: User):
         super().set_user(user)
         self.plugin_manager.hook.on_current_user_set(user=user, request=self)
 
+    def set_client_id(self, client_id: str):
+        """Overload TracimContext method to add plugin hook call."""
+        super().set_client_id(client_id)
+        self.plugin_manager.hook.on_current_client_id_set(client_id=client_id, request=self)
+
```

Comment:
I think we can define this method only on `TracimRequest` as it has a meaning only for HTTP requests coming from Tracim's frontend.